pkg/components/interface.go

```diff
@@ -25,7 +26,7 @@ type Component interface {
 	LoadConfig(*hcl.Body, *hcl.EvalContext) hcl.Diagnostics
 	// RenderManifests returns a map of Kubernetes manifests in YAML format, where
 	// the key is the file from which the content comes.
-	RenderManifests() (map[string]string, error)
+	RenderManifests() (*release.Release, error)
```

Comment:
- Since this method no longer renders manifests, we should give it a better name. How about `Release()` because it returns a `Release`?
- We should update the doc comment to correctly explain what the method does.